Makefile

```diff
@@ -392,7 +392,7 @@ all: $(alltests)
 examples : $(allexamples)
 
 # Benchmarks
-allbenchmarks = petsc-bp1 petsc-bp3
+allbenchmarks = petsc-bps
 bench_targets = $(addprefix bench-,$(allbenchmarks))
 .PHONY: $(bench_targets) benchmarks
 $(bench_targets): bench-%: $(OBJDIR)/%
```

Comment:
One downside to `make benchmarks` (exists currently as well) is that it runs against *all* backends, to include tmpl and memcheck. I don't think this is a huge issue, as you can run the benchmarks script at better granularity directly.

Reply:
Note, this is partially changed with `libceed_test.so`, so only the memcheck backend is run in the bench environment when it doesn't really make sense.